include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
@arstrube The current solution solves the problem and I cannot imagine how it can lead to a problem. All other proposals do not solve the real problem. So, unless there is a real solution, I won't be changing it.